### PR TITLE
Update the documentation regarding the migration to GitHub

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

File: magentoerpconnect/doc/conf.py [hunk 1]
```diff
@@ -19,20 +19,21 @@
...
 BASE_PATH = os.path.abspath(os.path.join(os.getcwd(), '../../..'))
 # You may need to change with your own paths
-ADDONS_PATHS = ('server/openerp/addons',
-                'webclient/addons',
-                'addons',
-                'openerp-connector',
-                'openerp-connector-ecommerce',
-                'e-commerce-addons',
-                'openerp-connector-magento')
+ADDONS_PATHS = ('odoo/openerp/addons',
```
Comment (pedrobaeza) [inline magentoerpconnect/doc/conf.py:29]: odoo for 7.0 version?

File: magentoerpconnect/doc/guides/key_questions.rst [hunk 2]
```diff
@@ -157,11 +157,11 @@ and the invoices entries for the 'automatic' workflows.
 However, you will still need to reconcile the bank entries.
 
 You may want to use the reconciliation modules of the
-branch `lp:banking-addons/bank-statement-reconcile/7.0`_.
+`bank-statement-reconcile`_ project.
 They are widely used in production and
 are specifically designed for the e-commerce.
 
-.. _`lp:banking-addons/bank-statement-reconcile/7.0`: https://code.launchpad.net/~banking-addons-team/banking-addons/bank-statement-reconcile-70
+.. _`bank-statement-reconcile`: https://github.com/OCA/bank-statement-reconcile
```
Comment (pedrobaeza) [inline magentoerpconnect/doc/guides/key_questions.rst]: Point to 7.0 branch?

File: magentoerpconnect/doc/guides/key_questions.rst [hunk 3]
```diff
@@ -173,13 +173,13 @@ the fiscal positions are not synchronized.
...
 `account_fiscal_position_rules` in the project
-`lp:openerp-fiscal-rules`_.
+`account-fiscal-rule`_.
 
 Note that this configuration will be done
 1 time in Magento and 1 time in OpenERP.
 But once the configuration is done, that works fine.
 
-.. _`lp:openerp-fiscal-rules`: https://code.launchpad.net/openerp-fiscal-rules
+.. _`account-fiscal-rule`: https://github.com/OCA/account-fiscal-rule
```
Comment (pedrobaeza) [inline magentoerpconnect/doc/guides/key_questions.rst]: Point to 7.0 branch?

File: magentoerpconnect/doc/conf.py (hunk 1 above)
Comment (guewen) [inline magentoerpconnect/doc/conf.py:29]: Yes: https://github.com/odoo/odoo/tree/7.0
Why?

File: magentoerpconnect/doc/guides/key_questions.rst (hunk 2 above)
Comment (guewen) [inline magentoerpconnect/doc/guides/key_questions.rst]: Changed

File: magentoerpconnect/doc/guides/key_questions.rst (hunk 3 above)
Comment (guewen) [inline magentoerpconnect/doc/guides/key_questions.rst]: Changed

File: magentoerpconnect/doc/conf.py (hunk 1 above)
Comment (pedrobaeza) [inline magentoerpconnect/doc/conf.py:29]: Ah, OK, you're right that the download folder is odoo.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**